custom_model_runner/datarobot_drum/resource/default_typeschema/model-metadata.yaml

```diff
@@ -0,0 +1,30 @@
+name: default-transform-model-metadata
```

Comment:
Delete the above init.py probably.

Reply:
I see what you're saying about this being a weird thing to put in the drum repo though.

Reply:
Nope, need it to include it in the wheel